### ISSUE TITLE
ember-cli/ext/promise deprecation warning; use rsvp instead.

### ISSUE THREAD

**Issue by rrenno** [achambers/ember-cli-deploy-git-revision-data#5]
Deprecation warning:

```
DEPRECATION: `ember-cli/ext/promise` is deprecated, use `rsvp` instead. Required here: 
  Object.<anonymous> (/Users/rrenno/zybooks/zybooks-web/node_modules/ember-cli-deploy-git-revision-data/index.js:6:19)
```

Fairly straightforward to fix it seems, I have a local branch that is working if you'd like.

**Comment by chrisgame** [comment]
Also found this and made a PR to fix it https://github.com/achambers/ember-cli-deploy-git-revision-data/pull/6

**Comment by achambers** [comment]
Closed in #6
